### DISCUSSION TITLE
Rustdesk display is not updating when worlking on other monitors

### DISCUSSION THREAD

**Discussion by anders-m** [rustdesk/rustdesk#4470] [Q&A]
When I have Rustdesk maximised on a monitor, showing the display of a computer in our Command Center, the Rustdesk client is not updating when I leave the monitor with my mouse and is working on my other monitors. The display is updating in the moment when I take the mouse pointer the that monitor again.
Is there a setting somewhere, that allows you to force updating the monitor all the time / at least every X sec?

**Comment by anders-m** [comment]
Anybody with some knowledge regarding the issue?

**Reply by rustdesk** [reply]
A video please.

**Reply by anders-m** [reply]
Here you go @rustdesk  :-)
[169265995 2023-06-06 12-40-18.zip](https://github.com/rustdesk/rustdesk/files/11663505/169265995.2023-06-06.12-40-18.zip)

**Reply by anders-m** [reply]
Can you see anything from the video?

**Comment by anders-m** [comment]
The latest version has solved the problem :-)
